aspects/schema.go

```diff
@@ -349,6 +359,37 @@ func (v *alternativesSchema) Validate(raw []byte) error {
 	return validationErrorf(sb.String())
 }
 
+// AtPath returns the list of schemas at the end of the path or an error if the
+// path cannot be followed.
+func (v *alternativesSchema) AtPath(path []string) ([]Schema, error) {
+	if len(path) == 0 {
+		return v.schemas, nil
+	}
+
+	var types []Schema
+	var lastErr error
+	for _, alt := range v.schemas {
+		altTypes, err := alt.AtPath(path)
+		if err != nil {
+			// some schemas may permit the path
+			lastErr = err
+			continue
+		}
+		types = append(types, altTypes...)
+	}
+
+	// TODO: find better way to combine errors
+	if len(types) == 0 {
+		return nil, lastErr
+	}
+
+	return types, nil
+}
+
+func (v *alternativesSchema) Type() SchemaType {
+	return Invalid
```

Comment:
here we could define a Alt type ?

Reply:
We can but I think it would just be changing the name. In the sense that we still don't know how to compare Alt and String so it's not a useful value. An `AtPath` called at a storageSchema will never "land" at an alternativeSchema so this would never be called and the path mismatch check shouldn't get an Invalid/Alt.

Reply:
same considerations